modules/rke2/master/instances_server.tf

```diff
@@ -110,7 +110,7 @@ resource "aws_instance" "master" {
     command = "echo ${aws_instance.master.public_ip} >/tmp/${var.resource_name}_master_ip"
   }
   provisioner "local-exec" {
-    command = "ssh-keyscan ${aws_instance.master.public_ip} > /root/.ssh/known_hosts"
+    command = "ssh-keyscan ${aws_instance.master.public_ip} > /$HOME/.ssh/known_hosts"
```

Comment:
Just a quick question -- this change to ```$HOME``` from ```root```, why has it been changed? I'm not necessarily following this change.

Reply:
its not hardcoding in root and also may be not a best practice assume that the user will have root priveleges

Reply:
The $HOME environment variable is almost always used as an alternative to hardcoding paths